### ISSUE TITLE
Dwain's Dashboard

### ISSUE THREAD

**Issue by pasi64** [dwainscheeren/dwains-lovelace-dashboard#752]
FR

Hi @ all,

today i tried to install this Dashboard on my HA.
The install documentation says, there will be, after restart, an auto generated Dashboard namend D.

But there is'nt anything.
What is going wrong?

**Comment by crpeck** [comment]
I am having the exact same issue. I even tried manually installing it, it is there under custom_components by it doesn't show up in the UI.

**Comment by dwainscheeren** [comment]
Did you tried a HA reboot and clear browser cache?

**Comment by crpeck** [comment]
Forgot about clearing browser, so I cleared all cookies & data from cache & restarted HA via the Developers section.  Still no dashboard, do you mean a complete reboot?

**Comment by crpeck** [comment]
It's bizarre, I used to have it installed then it broke, so I removed it.  I'm not sure how HACS works, but, I'm guessing everything for the Dashboard exists under custom_components & nowhere else?

**Comment by dwainscheeren** [comment]
Yes and a local folder for all the config. Did you check your HA error log during reboot what it says about DD?

**Comment by crpeck** [comment]
`[core-ssh config]$ grep -i dwain home-assistant.log
2023-12-11 08:03:31.844 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration dwains_dashboard which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant`
 
Also, it looked like you should create the configs directory, etc, on start?  Here's what is in the directory after a Restart of HA:
[core-ssh dwains_dashboard]$ ls -l
total 116
-rw-r--r--    1 root     root         55388 Dec 11 08:01 __init__.py
-rw-r--r--    1 root     root          1458 Dec 11 08:01 config_flow.py
-rw-r--r--    1 root     root            46 Dec 11 08:01 const.py
drwxr-xr-x    2 root     root          4096 Dec 11 08:01 js
-rw-r--r--    1 root     root          1203 Dec 11 08:01 load_dashboard.py
-rw-r--r--    1 root     root           444 Dec 11 08:01 load_plugins.py
drwxr-xr-x    3 root     root          4096 Dec 11 08:01 lovelace
-rw-r--r--    1 root     root           405 Dec 11 08:01 manifest.json
-rw-r--r--    1 root     root          7226 Dec 11 08:01 notifications.py
-rw-r--r--    1 root     root          8193 Dec 11 08:01 process_yaml.py
-rw-r--r--    1 root     root          3074 Dec 11 08:01 sensor.py
-rw-r--r--    1 root     root           878 Dec 11 08:01 services.yaml
drwxr-xr-x    2 root     root          4096 Dec 11 08:01 translations
[core-ssh dwains_dashboard]$ ls -ld
drwxr-xr-x    5 root     root          4096 Dec 11 08:01 .

**Comment by crpeck** [comment]
and now it magically works.

**Comment by ithaqua974** [comment]
> `[core-ssh config]$ grep -i dwain home-assistant.log 2023-12-11 08:03:31.844 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration dwains_dashboard which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant`
> 
> Also, it looked like you should create the configs directory, etc, on start? Here's what is in the directory after a Restart of HA: [core-ssh dwains_dashboard]$ ls -l total 116 -rw-r--r-- 1 root root 55388 Dec 11 08:01 **init**.py -rw-r--r-- 1 root root 1458 Dec 11 08:01 config_flow.py -rw-r--r-- 1 root root 46 Dec 11 08:01 const.py drwxr-xr-x 2 root root 4096 Dec 11 08:01 js -rw-r--r-- 1 root root 1203 Dec 11 08:01 load_dashboard.py -rw-r--r-- 1 root root 444 Dec 11 08:01 load_plugins.py drwxr-xr-x 3 root root 4096 Dec 11 08:01 lovelace -rw-r--r-- 1 root root 405 Dec 11 08:01 manifest.json -rw-r--r-- 1 root root 7226 Dec 11 08:01 notifications.py -rw-r--r-- 1 root root 8193 Dec 11 08:01 process_yaml.py -rw-r--r-- 1 root root 3074 Dec 11 08:01 sensor.py -rw-r--r-- 1 root root 878 Dec 11 08:01 services.yaml drwxr-xr-x 2 root root 4096 Dec 11 08:01 translations [core-ssh dwains_dashboard]$ ls -ld drwxr-xr-x 5 root root 4096 Dec 11 08:01 .

hello, I don't understand this part
I'm experiencing an issue with DD
her what my log says
![image](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/25313988/ac64cf47-6ccf-404e-ac5b-d527354aad3b)

**Comment by dwainscheeren** [comment]
Thanks for reporting this.

Dwains Dashboard 3.9.0 has now been released, and the source files for DD3 are available in this repository.

DD3 is no longer under active feature development and will slowly be phased out. Active development continues in Dwains Dashboard Next:
https://github.com/dwainscheeren/dwains-dashboard-next

At the time of writing, Dwains Dashboard Next is still in alpha.

If this issue is a feature request, please check whether it fits Dwains Dashboard Next instead.

If this issue is a bug, please retest with DD3 version 3.9.0. If the bug is still reproducible there, please open a new issue with:

- Home Assistant version
- Dwains Dashboard version
- browser and device
- browser console errors
- clear reproduction steps

For more context, see #906.

For that reason, this older issue is being closed as part of the DD3 issue tracker cleanup.
